src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -239,7 +239,8 @@
     </PropertyGroup>
     <Message Condition="$(tmpSonarIsTestFileByNameResult)=='true'" Importance="normal" Text="Sonar: ($(MSBuildProjectFile)) project is evaluated as a test project based on the project name" />
 
-    <Message Importance="normal" Text="Sonar: ($(MSBuildProjectFile)) Project categorized. SonarQubeTestProject=$(SonarQubeTestProject)" />
+    <Message Importance="normal" Condition="$(SonarQubeTestProject)=='false'" Text="Sonar: ($(MSBuildProjectFile)) Project categorized. SonarQubeTestProject=$(SonarQubeTestProject)." />
```

Comment:
now that you use the condition for `SonarQubeTestProject`, can we remove the `SonarQubeTestProject=$(SonarQubeTestProject).` from the logging ? Because this is just an internal variable of this MSBuild script... 

And it can also be a confusion point because the same name is used for variable in this script and for the MSBuild property added manually by users.

I suggest to use `Project categorized as MAIN project.` here